packages/genui/openui/styles/catalog/RadioGroup.css

```
@@ -0,0 +1,105 @@
+  gap: 18rpx;
+}
+
+.OpenUIRadioGroup-row .OpenUIRadioGroupContainer {
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 18rpx;
+  align-items: center;
+}
+
+.OpenUIRadioGroup-card .OpenUIRadioOption {
+  padding: 14rpx 16rpx;
```

Stylelint (17.14.0)

[error] Unknown unit "rpx" (unit-no-unknown)

(unit-no-unknown)